src/containers/Chat/ChatConversations/ChatConversations.tsx

```diff
@@ -114,6 +114,19 @@ export const ChatConversations: React.SFC<ChatConversationsProps> = () => {
     setSelectedIndex(-1);
   };
 
+  const handleChange = (e: any) => {
```

Comment:
Related to #157 . Put them both in the same branch for now since they're both related to search. Can split it later if I can't find a quick fix for real-time search/the fix is large.

Reply:
Yes, that's fine. Both can be in this branch.

Reply:
As discussed this is now part of separate PR https://github.com/glific/glific-frontend/pull/191